src/util.test.js

```diff
@@ -64,4 +82,89 @@ describe('util', () => {
       })).toEqual('LastName, FirstName MiddleName');
     });
   });
+
+  describe('getCheckoutSettings', () => {
+    it('should return undefined if input length is 0', () => {
+      expect(getCheckoutSettings([])).toEqual(undefined);
+    });
+
+    it('should return parsed JSON for a valid input', () => {
+      expect(getCheckoutSettings(settings)).toEqual({ prefPatronIdentifier: 'barcode,id' });
+    });
+
+    it('should return an empty object if there\'s an error', () => {
+      expect(getCheckoutSettings(emptySettings)).toEqual({});
+    });
+  });
+
+  describe('getPatronIdentifiers', () => {
```

Comment:
Could you cover this condition?
![image](https://user-images.githubusercontent.com/42437614/218499278-20ce9f12-fcdd-4a33-b54d-3d8a658f0dc4.png)